common/eth2/src/lighthouse/sync_committee_rewards.rs

```diff
@@ -0,0 +1,23 @@
+use serde::{Deserialize, Serialize};
+
+// Details about the rewards paid to sync committee members for attesting headers
+// All rewards in GWei
+
+#[derive(Debug, PartialEq, Clone, Serialize, Deserialize)]
+pub struct SyncCommitteeRewards {
+
+    pub execution_optimistic: Option<bool>,
+
+    pub finalized: Option<bool>,
+
+    pub data: Option<Vec<SyncCommitteeReward>> 
+}
+
+#[derive(Debug, PartialEq, Clone, Serialize, Deserialize)]
+pub struct SyncCommitteeReward {
+
+    pub validator_index: u64,
```

Comment:
We should encode this with quotes using `#[serde(with = "eth2_serde_utils::quoted_u64")]`